Transmission.xcodeproj/project.pbxproj

```diff
@@ -3022,15 +3038,15 @@
 			files = (
 			);
 			inputPaths = (
-				"third-party/miniupnpc/VERSION",
-				"third-party/miniupnpc/miniupnpcstrings.h.in",
+				"third-party/miniupnp/miniupnpc/VERSION",
+				"third-party/miniupnp/miniupnpc/miniupnpcstrings.h.in",
 			);
 			outputPaths = (
-				"third-party/miniupnpc/miniupnpcstrings.h",
+				"third-party/miniupnp/miniupnpc/src/miniupnpcstrings.h",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd third-party/miniupnpc\nsh updateminiupnpcstrings.sh \"$SCRIPT_INPUT_FILE_0\" \"$SCRIPT_INPUT_FILE_1\" \"$SCRIPT_OUTPUT_FILE_0\"\n";
+			shellScript = "cd third-party/miniupnp/miniupnpc\nsh updateminiupnpcstrings.sh\nmv miniupnpcstrings.h src/miniupnpcstrings.h\n";
```

Comment:
Wouldn't it fail? I think you should use `mv -f ...` here

Reply:
I mean this case.
1. You have built project once, this step executed once, `miniupnpcstrings.h` moved into place.
2. Changed either `third-party/miniupnp/miniupnpc/miniupnpcstrings.h.in` or `third-party/miniupnp/miniupnpc/VERSION` -> build step invalidated -> script run again.
3. File already present at the path -> `mv` fails.

Reply:
Well, it would succeed the first time and would not need any update until we upgrade miniupnp.
I've removed the `mv` entirely in fc7a420bb13d086be47af01255173b6fbc10bf11